stable/insights-agent/templates/rbac-reporter/rbac.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.rbacreporter.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-rbac-reporter-view
+  labels:
+    app: insights-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "insights-agent.fullname" . }}-rbac-reporter
```

Comment:
Once you delete the serviceaccount below, change this to `{{ include "insights-agent.fullname" . }}`